benchmarking/README.md

```diff
@@ -68,17 +68,19 @@ git clone https://github.com/sambanova/ai-starter-kit.git
 
 3. Update the `ai-starter-kit/.env` config file in the root repo directory. Here's an example:
     - Assume you have an endpoint with the URL
-        "https://api-stage.sambanova.net/api/predict/generic/12345678-9abc-def0-1234-56789abcdef0/456789ab-cdef-0123-4567-89abcdef0123"
+        "https://api-stage.sambanova.net/api/predict/generic/stream/12345678-9abc-def0-1234-56789abcdef0/456789ab-cdef-0123-4567-89abcdef0123"
 
     - You can enter the following in the env file (with no spaces):
 
     ``` bash
         SAMBASTUDIO_BASE_URL="https://api-stage.sambanova.net"
-        SAMBASTUDIO_BASE_URI="api/predict/generic"
+        SAMBASTUDIO_BASE_URI="api/predict/generic/stream"
```

Comment:
then could you please remove it from the readme as well

Reply:
good catch! thanks!